### DISCUSSION TITLE
Add Sonos controls to player provider entities

### DISCUSSION THREAD

**Discussion by dtspinella** [music-assistant/support#3344] [Feature requests and ideas]
I was wondering if it is possible to add Sonos control entities to the MA Sonos player provider such as:

Bass
Treble
Loudness
Status Light 
Volume Limit

I am not using the  Sonos integration from HA, I am just using the MA Sonos Player Provider and it would be useful to have these for Automations and Scripts.

**Comment by OzGav** [comment]
In the next beta an amazing addition is coming https://github.com/music-assistant/server/pull/1795

**Reply by dtspinella** [reply]
Very nice - I will be looking forward to this. Now if there was just a control for the status light…… 🙂

**Comment by patienttruth** [comment]
> In the next beta an amazing addition is coming https://github.com/music-assistant/server/pull/1795

Oh my my my!

**Comment by TGrounds** [comment]
I just added Music Assistant to my Home Assistant, and I really love how it integrates and works with everything - especially the automations.  I would love to clean up my Home Assistant environment by removing the Sonos integration and use the Sonos integration via Music Assistant completely and by adding these additional entities then we will have parity and will enable me to do that.  It may be worth digging through the Sonos integration to see anything in the background that may be valuable?
